compose/ui/ui/src/jvmMain/kotlin/androidx/compose/ui/node/SortedSet.jvm.kt

```diff
@@ -17,5 +16,5 @@
 
 package androidx.compose.ui.node
 
-internal actual class TreeSet<E> actual constructor(comparator: Comparator<in E>) :
+internal actual class SortedSet<E> actual constructor(comparator: Comparator<in E>) :
```

Comment:
Can it be a typealias in this case?

Reply:
It doesn't work now because of `internal` visibility